test/gtest/uct/test_uct_ep.cc

```diff
@@ -186,13 +178,15 @@ UCS_TEST_SKIP_COND_P(test_uct_ep, destroy_entity_after_send,
     const unsigned max_iter = 300 / ucs::test_time_multiplier();
 
     for (unsigned i = 0; i < max_iter; ++i) {
-        create_sender();
-        connect();
+        create_entity(0);
```

Comment:
I aligned inplementation with existing in UCP, changing of this approach will blow up the patch even more but I also dont like much positioning sender and receiver in vector, it could be a next step of refactoring if needed.

Reply:
No big changes needed)
It should normally decrease the size of the patch, because you can do assignment in 1 line instead of 2.
I'm talking about this change that you have made:
```
-    uct_test::entity *
+    void
    create_entity(size_t rx_headroom, uct_error_handler_t err_handler = NULL,
```
My understanding is that `create_entity` was already returning the newly created object, but it does not anymore. That's why yo uhave to lookup newly created object inside the container, right? So I'm not sure why that changed. But if we use previous signature, then assignment can be done in one line, right?

Reply:
got your point 👍, fixed